jsonschema2dj/management/custom_field_validators.py

```diff
@@ -14,12 +12,11 @@ def __init__(self, factor, message=None):
             self.message = message
 
     def __call__(self, value):
-
-        if isinstance(factor, float):
-            quotient = value / factor
+        if isinstance(self.factor, float):
```

Comment:
you dont need any of these lines. This is a hang over from Python2 days where the divide operator was a bit weird but dont care about that any more 

this is sufficient 
```python
if value % self.factor == 0:
    raise ValidationError(...)
```